bin/mail/2016-08-09-hoc-latam/send-real-emails

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+#3661
```

Comment:
Yeah, it's an upper limit. I gave some padding in case we have more contacts since my test query (from the 2016 signup) when I actually run the scripts.